subsys/bluetooth/controller/include/ll.h

```diff
@@ -206,3 +206,38 @@ void ll_rx_mem_release(void **node_rx);
 void ll_timeslice_ticker_id_get(uint8_t * const instance_index, uint8_t * const user_id);
 void ll_radio_state_abort(void);
 uint32_t ll_radio_state_is_idle(void);
+
+/* Direction Finding */
+
+/* @brief Function sets CTE transmission parameters for a connection.
+ *
+ * @param[in]handle			Connection handle.
+ * @param[in]cte_types			Bitfield holding information about
+ *					allowed CTE types.
+ * @param[in]switching_patterns_len	Numbe of antenna IDs in switching
+ *					pattern.
+ * @param[in]ant_id			Array od antenna IDs that are antenna
+ *					switching pattern.
+ *
+ * @return Status of command completion.
+ */
+uint8_t ll_df_set_conn_cte_tx_params(uint16_t handle, uint8_t cte_types,
+				     uint8_t switching_patterns_len,
+				     uint8_t *ant_id);
+
+/* @brief Function provides information about Direction Finding
+ * antennae switching and sampling related settings.
+ *
+ * @param[out]switch_sample_rates       Pointer to store available antennae
+ *                                      switch-sampling configurations.
+ * @param[out]num_ant                   Pointer to store number of available
+ *                                      antennae.
+ * @param[out]max_switch_pattern_len    Pointer to store maximum number of
+ *                                      antennae switch patterns.
+ * @param[out]max_cte_len               Pointer to store maximum length of CTE
+ *                                      in [8us] units.
+ */
```

Comment:
Move detailed documentation into ull_df.c file, and keep a one line comment here in this internal private header file

Reply:
Why do you like to move it into ull_df.c file. It seems to me that it is common to have functions documentation in header files. Isn't it?

Reply:
Take a look in the bluetooth subsystem, most of internal headers have brief onliner. As a developer, our focus is on the definition of the function in the c file. Doxygen documentation makes sense for public API where users are focused on API usage.

Reply:
I've moved function headers to ull_df.c as requested but... I steel feel that correct place for such documentation is in header files. In my humble opinion this is a common place for it even functions are only internal. I see such functions (global functions) as an API of a module/submodule. 
I'm new here and and I steel feel lack lack of such information is uncomfortable, especially in case of some more complicated functions. 
I've added it here and also in all files I have created following the scouts rule: leave the campground cleaner than you found it.